src/pages/NewChatConfirmPage.tsx

```diff
@@ -92,51 +81,69 @@ function NewChatConfirmPage({newGroupDraft, allPersonalDetails}: NewChatConfirmP
         if (!newGroupDraft) {
             return;
         }
-        const newSelectedParticipants = newGroupDraft.participants.filter((participant) => participant.login !== option.login);
-        Report.setGroupDraft(newSelectedParticipants);
+        const newSelectedParticipants = (newGroupDraft.participants ?? []).filter((participant) => participant.login !== option.login);
+        Report.setGroupDraft({participants: newSelectedParticipants});
     };
 
     const createGroup = () => {
         if (!newGroupDraft) {
             return;
         }
-        const logins: string[] = newGroupDraft.participants.map((participant) => participant.login);
-        Report.navigateToAndOpenReport(logins, true, groupName);
+
+        const logins: string[] = (newGroupDraft.participants ?? []).map((participant) => participant.login);
+        Report.navigateToAndOpenReport(logins, true, newGroupDraft.reportName ?? '', newGroupDraft.avatarUri ?? '', fileRef.current);
     };
 
     const navigateBack = () => {
         Navigation.goBack(ROUTES.NEW_CHAT);
     };
 
+    const navigateToEditChatName = () => {
```

Comment:
both `navigateToEditChatName` and `navigateBack` can be moved outside the component